.github/workflows/pr.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           git fetch origin
           export NUM_RELEASES=2 # only 2 last tags to save build time 
           bash makedoc.sh development
-          netlify deploy --dir=_build/dirhtml --alias="ft-${{ env.BRANCH_NAME }}" --message="Deploying docs to ${{ env.BRANCH_NAME }} branch"
+#          netlify deploy --dir=_build/dirhtml --alias="ft-${{ env.BRANCH_NAME }}" --message="Deploying docs to ${{ env.BRANCH_NAME }} branch"
```

Comment:
is this commented out?

Reply:
the token is invalid, we need to get a new token, Nan is helping us to do so @gmastrapas